core/src/main/java/io/ddf/DDF.java

```diff
@@ -83,6 +85,20 @@ public void setIsDDFView(boolean isDDFView) {
     this.mIsDDFView = isDDFView;
   }
 
+  public boolean isCache() {
+    return this.isCache;
+  }
+
+  public void cache() {
+    this.isCache = true;
+    this.getRepresentationHandler().cache(false);
```

Comment:
Also, in `cache` and `uncache` methods, the flag `isCached` always changes the value upon calling, should only change this after the operation finishes successfully?